src/config/settings.py

```diff
@@ -5,6 +5,8 @@
 import environ
 from django.core.management.utils import get_random_secret_key
 
+from ..apps.bot.logger import create_log_directory
```

Comment:
Импорт должен быть явным.
```suggestion
from src.apps.bot.logger import create_log_directory
```

Reply:
@AlanSosnovsky Сейчас деплой падает, необходимо поправить ошибку перед мержем.